src/main/java/org/dependencytrack/resources/v1/TagResource.java

```diff
@@ -146,6 +146,36 @@ public Response deleteTags(
         return Response.noContent().build();
     }
 
+    @POST
```

Comment:
The API has historically used `PUT` for creating resources, and `POST` for updating them. Whether that is best practice is probably debatable, but for the sake of consistency, let's use `PUT` here as well.

Reply:
Thanks for the clarification. Will change this for consistency.